superset-frontend/cypress-base/cypress/e2e/dashboard/drillby.test.ts

```diff
@@ -588,7 +719,7 @@ describe('Drill by modal', () => {
     it('Gauge Chart', () => {
       testEchart('gauge_chart', 'Gauge Chart', [
         [151, 95],
-        [300, 143],
+        [300, NaN], // this fixes the test after upgrading to Cypress 13 
```

Comment:
This is completely voodoo magic to me. Previous coordinates didn't work and frustratingly I entered `NaN` and it worked